test/test_cfgh.py

```diff
@@ -79,15 +80,15 @@ def test_add_config_that_exists(self, config_handler: ConfigHandler):
         """ Check that file path was added """
         assert temp_file_path in config_handler._cfg
 
-    def test_add_config_that_not_exists(self, capsys: CaptureFixture, config_handler: ConfigHandler):
+    def test_add_config_that_not_exists(self, caplog: LogCaptureFixture, config_handler: ConfigHandler):
         """ Set non existent path """
         non_existent_config_path: str = "/path/to/non/existent/config.cfg"
         """ add non existent path """
```

Comment:
BTW, why you use docstrings as commends? This is not really good. It should be:

```python
def test_something():
    """
    This test tests something
    """
    # Set things
    a:str = "bs"
    
    # Add something
    b:str = "bs bs"
```

...etc. Could you please also clean that up in this PR still? (everywhere).

Reply:
I have no idea why they're there 
I just refactored the tests without really looking at them

But I can cleanup yes